### ISSUE TITLE
Placing a view to block to basic page throws 404

### ISSUE THREAD

**Issue by alesbencina** [upchuk/facets_pretty_paths#9]
Hello, 

I found out that when we place a block with view (viewblock) and place it to basic page we get 404 error.

Infact the generated URL seems to be fine (/products/category/taxonomy-name-4).

**Comment by upchuk** [comment]
Yeah, this is a duplicate of https://www.drupal.org/project/facets_pretty_paths/issues/3012910. Closing here.
